tests/functional/z_cluster/test_scale_mon_pods.py

```
@@ -0,0 +1,387 @@
+        provis_pods = pod.get_pods_having_label(
+            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
+            config.ENV_DATA["cluster_namespace"],
+        )
+        provis_pods += pod.get_pods_having_label(
+            constants.CSI_RBDPLUGIN_PROVISIONER_LABEL,
+            config.ENV_DATA["cluster_namespace"],
+        )
+        provis_pod_names = [p["metadata"]["name"] for p in provis_pods]
+
+        # Maintenance of the node (unschedule and drain)
+        log.info(f"Chosen nodes for draining are {mon_nodes[0:2]} ")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)